Sources/ContainerClient/Core/ClientHealthCheck.swift

```diff
@@ -43,6 +43,16 @@ extension ClientHealthCheck {
         guard let apiServerCommit = reply.string(key: .apiServerCommit) else {
             throw ContainerizationError(.internalError, message: "failed to decode apiServerCommit in health check")
         }
-        return .init(appRoot: appRoot, installRoot: installRoot, apiServerVersion: apiServerVersion, apiServerCommit: apiServerCommit)
+        // Optional fields for newer servers
+        let apiServerBuild = reply.string(key: .apiServerBuild)
```

Comment:
You can assume server and client are tightly coupled for now.

That is to say, go ahead and guard these assignments just like the others.